orchagent/notifier.h

```diff
@@ -2,8 +2,8 @@
 
 class Notifier : public Executor {
 public:
-    Notifier(NotificationConsumer *select, Orch *orch)
-        : Executor(select, orch)
+    Notifier(NotificationConsumer *select, Orch *orch, const std::string& notifier_name)
```

Comment:
>notifier_name [](start = 74, length = 13)

I notice there is a similar name at NotificationConsumer.m_channel. I prefer using that one instread of a new parameter.

Reply:
Renamed to m_channel